code/game/objects/items/devices/modkit.dm

```diff
@@ -147,3 +147,22 @@
 	parts[1] =	1
 	original[1] = /obj/item/weapon/gun/energy/gun
 	finished[1] = /obj/item/weapon/gun/energy/gun/nuclear
+
+/obj/item/device/modkit/plasmacutter
+	name = "plasma cutter conversion kit"
+	desc = "A set of tools that enables conversion of a mining diamond drill into a plasma cutter. Needs to be loaded with the parts of a proto-kinetic accelerator first."
+
+/obj/item/device/modkit/plasmacutter/New()
+	..()
+	parts = new/list(1)
```

Comment:
I can't really ask you to change the whole file for your update, but I'm not really sure why those are lists

Reply:
I think deity made those so you can ask him. My guess: ability to make a modkit have different types of parts for different modifications

Reply:
This is also creating a list with a first element of 1, not a list of length 1, but whatever